optimize/backend/src/it/java/io/camunda/optimize/AbstractCCSMIT.java

```diff
@@ -338,11 +362,34 @@ protected OffsetDateTime getTimestampForZeebeAssignEvents(
         Instant.ofEpochMilli(startOfElement.getTimestamp()), ZoneId.systemDefault());
   }
 
+  protected OffsetDateTime getTimestampForZeebeLastAssignEvents(
```

Comment:
```suggestion
  protected OffsetDateTime getTimestampForZeebeLastAssignedEvents(
```
Just for correctness to align it with the intent